src/main/java/framework/user_screen/UserLogUI.java

```diff
@@ -117,7 +101,17 @@ private void createScreenComponents() {
         userPassWordField = new JPasswordField();
         RegisterButton = new JButton("Register");
         LoginButton = new JButton("Login");
-        forgetPassword = new JButton("Forget Password");
+
+    }
+    // Temporary test file
+    public static void main(String[] args) {
+        UserLoginDataBaseGateway dataBaseGateway = new UserLoginFileGateway();
+        UserLogViewModel viewModel = new UserLogViewModel();
+        UserStatusViewModel statusViewModel = new UserStatusViewModel();
+        UserLogOutputBoundary presenter = new UserLogPresenter(viewModel, statusViewModel);
+        UserLogInputBoundary useCase = new UserLogUseCase(presenter, dataBaseGateway);
+        UserLogController controller = new UserLogController(useCase);
+        new UserLogUI(controller, viewModel);
     }
```

Comment:
This file is the old version of the UserLogUI, please preserve the newer version in Main Branch